source/guide/3.2/guide-manager-watchdog.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: bt_wiki
+title: Cloudify Manager Watchdog Guide
+category: Guides
+publish: true
+abstract: "Guide for Cloudify Manager Watchdog"
+pageord: 900
+---
+{%summary%}{{page.abstract}}{%endsummary%}
+
+
+
+{%note title=Note%}
+The Cloudify manager watchdog is a premium feature, available only in the commercial version of Cloudify.
+{%endnote%}
+
+
+# Overview
+
+The Cloudify manager watchdog is a long-running Python process which monitors a Cloudify manager, and attempts to revive it if it goes down or is unresponsive.
+
+
+# Usage
+
+The manager watchdog may run in any environment which contains the requirements for performing recovery of the manager - meaning that it requires connectivity to the manager machine and the private key file used to connect to the manager's host (The key will be used to connect to the new host, which will be assigned with the same public key as the original host, in a recovery scenario).
+
+{%info title=Information%}
+The manager watchdog is agnostic to Cloudify's security features, in the sense that it'll use them transparently when they're configured, without any additional setup or configuration to the watchdog itself.
+{%endinfo%}
+
+The manager watchdog is operated via the command-line program `cfy-watchdog`. Mandatory arguments are the monitored manager's IP and the path to the private key file.
+
+{%note title=Note%}
+When attempting recovery of the manager, the watchdog will run a local workflow similar to the one run during bootstrap. It is therefore needed for any plugins required by the manager-blueprint which was used in bootstrap to be installed in the manager watchdog's environment as well.
```

Comment:
i think its worth stating here that the watchdog actually runs the cfy recover command, which in turn runs the local workflow 'heal'